roles/java_installer/tasks/main.yml

```diff
@@ -1,19 +1,19 @@
 - name: "Amazon Linux Java Install"
   ansible.builtin.set_fact:
-    java_package: java-17-amazon-corretto
+    java_package: java-21-amazon-corretto
   when: ansible_distribution == "Amazon"
 
 - name: "Debian Java Install"
   ansible.builtin.set_fact:
-    java_package: openjdk-17-jdk
+    java_package: openjdk-21-jdk
```

Comment:
Also, it seems Debian still doesn't have this package. It installs fine on Amazon, though.

Reply:
Yeah, interestingly debian doesn't have 21 in the stable packages still and even testing onlly has 22 and 23..

Reply:
I would say to revert this line then, so we can still technically run against Debian.

Reply:
Updated.